### DISCUSSION TITLE
Do I need change config.ini always?

### DISCUSSION THREAD

**Discussion by Atorcha** [Louisvdw/dbus-serialbattery#954] [Q&A] [answered]
Hi. First of all say that thanks for your job. Ot help us a lot. Then my question is. 

I have different parameters in my JKBMS, for example max current discharge 150A. What will happend then?

thanks.

**Comment by zigomatichub** [comment, marked as answer]
Discharge max 150A is in JKBMS and with another value higher in dbus-serialbattery result in max discharge of 150A as your BMS is controlling the max current discharge.
Discharge max 150A is in JKBMS and with another value lower in dbus-serialbattery result in max discharge of value lower in your ESS / Victron system. If for some reasons, even with let's say 100A in dbus and effectively it goes to 140A, your BMS will keep deliver until 150A.
